frontend/src/container/QueryBuilder/filters/HavingFilter/HavingFilter.tsx

```diff
@@ -198,6 +198,18 @@ export function HavingFilter({
 		resetChanges();
 	};
 
+	const handleBlur = useCallback((): void => {
+		if (searchText) {
+			const isValidSearch = isValidHavingValue(searchText);
```

Comment:
Just confirm once that the only case when we want this blur event to trigger is when we have searchText right? Rest lgtm